cmd/gh-classroom/clone/student-repos/student-repos.go

```diff
@@ -101,6 +101,13 @@ func NewCmdStudentRepo(f *cmdutil.Factory) *cobra.Command {
 					fmt.Printf("Skip existing repo: %v use gh classroom pull to get new commits\n", clonePath)
```

Comment:
nit: even if we skip all repos, we will still get something like:
```
Skip existing repo: ...
Skip existing repo: ...
Cloned 2 repos.
```

Reply:
Nice catch :) Updated the logging to not count repos that were skipped!